assets/source/setup-guide/app/data/settings/actions.js

```diff
@@ -40,6 +40,13 @@ export function setIsUpdating( isUpdating ) {
 	};
 }
 
+/**
+ * Update the settings in the store or in DB
+ *
+ * @param {Object} data The data to be updated
+ * @param {boolean} saveToDb If true it persists the changes in DB
```

Comment:
💅  It'd be more precise to indicate `saveToDb` is an optional parameter.

```suggestion
 * @param {boolean} [saveToDb=false] If true it persists the changes in DB
```